velox/connectors/hive/HiveDataSource.h

```diff
@@ -107,7 +107,7 @@ class HiveDataSource : public DataSource {
   dwio::common::ReaderOptions readerOpts_;
   memory::MemoryPool* pool_;
   VectorPtr output_;
-  RowTypePtr readerOutputType_;
+  RowTypePtr outputType_;
```

Comment:
`outputType_` is not a proper name, `readerOutputType_` is data source / table scan output + remaining filter inputs, I would not change this name